test/00-basic.test.ts

```diff
@@ -0,0 +1,33 @@
+import * as supertest from 'supertest';
+import { expect } from 'chai';
+const fixturePath = __dirname + '/fixtures/00-basic/config';
+import { testInit, testDeInit, testLocalServer } from './lib/test-init';
+
+describe('00 basic tests', function () {
+	let pineServer: any;
```

Comment:
If you remove the `: any` does it infer the type correctly?
```suggestion
	let pineServer;
```
If not then you can also do
```suggestion
	let pineServer: ReturnType<typeof testInit>;
```

Reply:
It will infer an implicit any type, so that I've typed it as:
`Awaited<ReturnType<typeof testInit>>`